src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/does-live-with-spouse/doesLiveWithSpouse.js

```diff
@@ -47,46 +32,24 @@ export const uiSchema = {
     'ui:errorMessages': { required: 'Please give a brief explanation' },
   },
   currentSpouseAddress: {
-    'ui:title': 'Your spouse’s address',
-    'ui:options': {
-      expandUnder: 'spouseDoesLiveWithVeteran',
-      expandUnderCondition: false,
-    },
-    currentSpouseCountry: {
-      'ui:title': 'Country',
-      'ui:required': doesLiveTogether,
-      'ui:errorMessages': { required: 'Please select a country' },
-    },
-    currentSpouseStreet: {
-      'ui:title': 'Street',
-      'ui:required': doesLiveTogether,
-      'ui:errorMessages': { required: 'Please enter a street address' },
-    },
-    currentSpouseLine2: {
-      'ui:title': 'Line 2',
-    },
-    currentSpouseLine3: {
-      'ui:title': 'Line 3',
-    },
-    currentSpouseCity: {
-      'ui:title': 'City',
-      'ui:required': doesLiveTogether,
-      'ui:errorMessages': { required: 'Please enter a city' },
-    },
-    currentSpouseState: {
-      'ui:title': 'State',
-      'ui:required': doesLiveTogether,
-      'ui:errorMessages': {
-        required: 'Please enter a state, or country if outside the U.S.',
-      },
-    },
-    currentSpousePostalCode: {
-      'ui:required': doesLiveTogether,
+    ...addressUISchema(false, 'currentSpouseAddress', doesLiveTogether),
+    ...{
```

Comment:
I _think_ the `'ui:title'` and `'ui:options'` can be moved directly under the `...addressUISchema(...)` line. So we don't need the `...{}` wrapper